Makefile

```diff
@@ -602,6 +602,14 @@ generate_yaml:
 $(HELM):
 	bin/init_helm.sh
 
+# create istio-remote.yaml
+istio-remote.yaml: $(HELM)
+	cat install/kubernetes/templates/namespace.yaml > install/kubernetes/$@
+	$(HELM) template --namespace=istio-system \
+		  --set global.pilotIp="pilotIpReplace" \
```

Comment:
Should it be a variable ? Not sure how this 'replace' will work. 

See my comments on the doc - using pod IP doesn't really work, we need the ILB address or a DNS name.

Reply:
The folks doing CI for this work had asked for something they could regex in the unit tests - that is what this is.  I think a better approach is possibly to import variables "pilotEndpoint, mixerEndpoint" - and set them to pilotEndpointReplace.

Cheers
-steve

Reply:
ILB would be great, but at least for e2e testing we can check the IP that is assigned to the POD at run time.   So at least for e2e tests this is workable as is.

Reply:
Can you regex/replace {{.Values.pilotIp}} ? 

We can refine it in separate PRs of course, not holding it on this.